### ISSUE TITLE
What should I do about Firebase?

### ISSUE THREAD

**Issue by samueljseay** [Leanplum/Leanplum-Segment-Android#2]
I'm finding it awkward that the initialisation of Leanplum doesn't provide the option to enable Leanplum's support of firebase messaging via:

```java
LeanplumPushService.enableFirebase();
```
Which appears to need to happen before `Leanplum.start()` 

Is it possible to provide this functionality somehow in the integration? I guess it would have to be configurable. I assume this is a setting that would live in the Segment UI for this integration?

**Comment by benmarten** [comment]
Yes you are right, this need to happen before `Leanplum.start()`. 
At the moment I am not sure if it wouldn't just be possible to call this line here: https://github.com/Leanplum/Leanplum-Segment-Android/blob/master/Example/src/main/java/com/leanplum/segment/example/MainActivity.java#L26

If not then yes we have to see how we make it configurable, maybe via the Segment options...
Feel free to give it a try, I will tomorrow...

**Comment by samueljseay** [comment]
Thanks, yes thats a good point regarding calling it before setting up integrations. We are going to give that a go first. It'd be a nice to have to wrap up all the integration work in the segment integration but if this works it isn't essential.
